README.md

```diff
@@ -16,6 +18,11 @@ Castle Core is &copy; 2004-2017 Castle Project. It is free software, and may be
 
 ## Building
 
+| Platform | Build Status |
+|----------|-------------:|
+| Windows  | [![Build status](https://ci.appveyor.com/api/projects/status/49a6i0ydiku56r5b/branch/master?svg=true)](https://ci.appveyor.com/project/castleproject/core) |
+| Linux    | [![Build Status](https://travis-ci.org/castleproject/Core.svg?branch=master)](https://travis-ci.org/castleproject/Core) |
```

Comment:
Since Travis also shows pull requests on the main page we should link to a different page, probably this one:
https://travis-ci.org/castleproject/Core/branches